llpc/tool/llpcPipelineBuilder.cpp

```diff
@@ -84,19 +84,24 @@ bool PipelineBuilder::isGraphicsPipeline() const {
 //
 // @returns : Result::Success on success, other status on failure.
 Result PipelineBuilder::build() {
-  BinaryData pipeline = {};
+  llvm::SmallVector<BinaryData, 1> pipelines;
+  pipelines.push_back(BinaryData());
+  pipelines[0] = {};
```

Comment:
The assignment redundant, the first push back will already initialize the element. I'd simplify this to:
```
pipelines.push_back({});
```